src/styles/blocks/header.scss

```
@@ -0,0 +1,54 @@
+.header_container {
```

`header_container` it's a bem element of `header`, so move it to corresponding block
```suggestion
.header__container {
```